activefires_pp/post_processing.py

```diff
@@ -727,6 +741,16 @@ def add_unique_day_id(self, afdata):
         afdata['detection_id'] = id_list
         return afdata
 
+    def add_tb_celcius(self, data_frame):
+        """Add a column with TB in Celcius to the fire detection data frame."""
+        tbc_list = []
+        for _i in range(len(data_frame)):
```

Comment:
I'm really surprised we have to loop on the data frame here. This is known to be slow in pandas, and vectorized operations should be available.

Reply:
Yes, Agree, however, have no clue how to this in a vectorized manner with Pint!?

Reply:
Fixed.